onnxruntime.sh

```diff
@@ -44,8 +166,6 @@ mkdir -p "$INSTALLROOT/etc/modulefiles"
 MODULEFILE="$INSTALLROOT/etc/modulefiles/$PKGNAME"
 alibuild-generate-module --lib > "$MODULEFILE"
 cat >> "$MODULEFILE" <<EoF
-
 # Our environment
-set ${PKGNAME}_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
-prepend-path ROOT_INCLUDE_PATH \$${PKGNAME}_ROOT/include/onnxruntime
```

Comment:
why was this taken out? I believe this change leads to crashes of all MC jobs since 4 days. See here: https://github.com/alisw/alidist/pull/5793